src/react/workflow/ReplicationForm.tsx

```diff
@@ -196,22 +187,22 @@ function ReplicationComponent({
                     }
 
                     return (
-                      <Select
-                        id="sourceBucket"
-                        onChange={onChange}
-                        options={options}
-                        formatOptionLabel={renderSource(locations)}
-                        isDisabled={isEditing}
-                        isOptionDisabled={(option) => option.disabled === true}
-                        value={result}
-                      />
+                      <Select value={result} onChange={onChange} isDisabled={isEditing}>
+                        {options &&
+                          options.map((o, i) => (
+                            <Option title={o.disabled ? 'Versioning is disabled' : ''} key={i} value={o.value}
```

Comment:
I am wondering do we need the `title` for each of the options?

Reply:
Well, we need the title with the value  "Versioning is disabled" just in case of a disabled option